mne/minimum_norm/inverse.py

```diff
@@ -629,21 +628,50 @@ def prepare_inverse_operator(orig, nave, lambda2, method, verbose=None):
 
 @verbose
 def _assemble_kernel(inv, label, method, pick_ori, verbose=None):
-    """Assemble the kernel."""
-    #
-    #   Simple matrix multiplication followed by combination of the
-    #   current components
-    #
-    #   This does all the data transformations to compute the weights for the
-    #   eigenleads
-    #
+    """Assemble the kernel.
+
+    Simple matrix multiplication followed by combination of the current
+    components. This does all the data transformations to compute the weights
+    for the eigenleads.
+
+    Parameters
+    ----------
+    inv : instance of InverseOperator
+        The inverse operator to use. This object contains the matrices that
+        will be multiplied to assemble the kernel.
+    label : Label | None
+        Restricts the source estimates to a given label. If None,
+        source estimates will be computed for the entire source space.
+    method : "MNE" | "dSPM" | "sLORETA"
+        Use mininum norm, dSPM or sLORETA.
+    pick_ori : None | "normal" | "vector"
+        Which orientation to pick (only matters in the case of 'normal').
+
+    Returns
+    -------
+    K : 2D array (n_vertices, n_channels)
```

Comment:
do you prefer `array, shape (n, m)` or `array of shape (n, m)`, or doesn't it matter?